### ISSUE TITLE
UTF-16 stuck

### ISSUE THREAD

**Issue by Dakta** [MacIrssi/MacIrssi#5]
Once set to UTF-16, the character encoding can no longer be changed via the Preferences GUI, and must be reset from the config (which should really be in the preferences file).

**Comment by ghost** [comment]
Yep, my bad. I know about this one I just haven't had time to fix it yet. On the note of the config file, it would be possible to move it into the preferences file but then it wouldn't be compatible with a standard irssi config and would break anybody who shares their Irssi configs across multiple OS's.

**Comment by ghost** [comment]
You can probably also fix it by issuing "/set term_charset utf-8" at the prompt.

**Comment by ghost** [comment]
Fixed in e413e41556f8765cde45d54e5a5effa1ab48668e

**Comment by zyke** [comment]
I am having the exact problem with my macirssi. 
I downloaded it today and accidentally changed the character settings to UTF-16. now I cant change back. I tried to use the "/set term_charset utf-8" at the status window, but without results. I checked the e413e41 solution as well, but I could not figure out where to find the source files.

I also tried to uninstall and reinstall the application, though without luck.

Would appreciate some help.
Thanks
